src/test/resources/features/actions/line_mover.feature

```diff
@@ -1,44 +1,59 @@
-    <rule_like1> NAME1:
-        <rule_section1>: <content>
+    <rule_like1> <name1>:
+        <rule_section1>:
+            <content1>
+    <rule_like2> <name2>:
+        <rule_section2>: <content2>
+    """
+    When I put the caret at <rule_like1> <name1>:
+    And I invoke MoveStatementDown action
+    Then editor content will be
+    """
+    <rule_like2> <name2>:
+        <rule_section2>: <content2>
+    <rule_like1> <name1>:
+        <rule_section1>:
+            <content1>
     """
     Examples:
-      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content         |
-      | rule        | rule        | input          | output         | "file.txt"      |
-      | checkpoint  | rule        | input          | output         | "file.txt"      |
-      | subworkflow | rule        | workdir        | output         | "file.txt"      |
-      | rule        | checkpoint  | input          | output         | "file.txt"      |
-      | checkpoint  | checkpoint  | input          | output         | "file.txt"      |
-      | subworkflow | checkpoint  | workdir        | output         | "file.txt"      |
-      | rule        | subworkflow | input          | workdir        | "file.txt"      |
-      | checkpoint  | subworkflow | input          | workdir        | "file.txt"      |
-      | subworkflow | subworkflow | workdir        | workdir        | "file.txt"      |
-
-  Scenario Outline: Permutation up between rule and comment sections
+      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content1    | name1 | name2   | content2   |
+      | rule        | rule        | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | checkpoint  | rule        | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | subworkflow | rule        | workdir        | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | rule        | checkpoint  | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | checkpoint  | checkpoint  | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | subworkflow | checkpoint  | workdir        | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | rule        | subworkflow | input          | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | checkpoint  | subworkflow | input          | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | subworkflow | subworkflow | workdir        | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
+      | rule        | def         | input          | n              | "file.txt"  | NAME1 | NAME2() | int        |
+      | checkpoint  | def         | input          | n              | "file.txt"  | NAME1 | NAME2() | int        |
+      | subworkflow | def         | workdir        | n              | "file.txt"  | NAME1 | NAME2() | int        |
```

Comment:
Test data doesn't match name of the test, test ` Permutation up/down between rules sections` is supposed to be about rule sections swap, but `def` isn't `rule like` element and isn't rule section. Please split test in two or use better names for the test and test args to make test data matching test name.

Reply:
I added different test for python functions